presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -39,6 +39,9 @@
 })
 public class FeaturesConfig
 {
+    private double cpuCostWeight = 0.75;
```

Comment:
More like educated guesses based on what I have seen during TPCH or TPCDS benchmarks/PoCs/customer problems that reached me. But without any deep analysis what would be the best.